docs/cifar10_examples.md

```diff
@@ -0,0 +1,18 @@
+# CIFAR10 examples
+
+The CIFAR-10 dataset is a collection of images that are commonly used to train machine learning and computer vision algorithms. It is one of the most widely used datasets for machine learning research. The CIFAR-10 dataset contains 60,000 32x32 color images in 10 different classes. There are 6,000 images of each class. Thus, we use CIFAR10 as example to introduce using NNI in the field of computer vision.
```

Comment:
shall we introduce the search space and tell the user which parameter we want to tuner in this example?